### PR TITLE
Bump FP base price

### PR DESCRIPTION
**Pull request by velzevur**
PT [SC FP minimum base gas bump](https://www.pivotaltracker.com/story/show/166001363)

### CODE REVIEW

**Comment by velzevur** [comment]
Once we reach agreement on the multiplier, I will prepare a corresponding `protocol` PR

File: apps/aecore/src/aec_governance.erl [hunk 1]
```diff
@@ -84,29 +85,38 @@ block_mine_reward(Height) when is_integer(Height), Height > 0 ->
...
+tx_base_gas(spend_tx, _)                  -> ?TX_BASE_GAS;
+tx_base_gas(channel_deposit_tx, _)        -> ?TX_BASE_GAS;
+tx_base_gas(channel_close_mutual_tx, _)   -> ?TX_BASE_GAS;
+tx_base_gas(channel_close_solo_tx, _)     -> ?TX_BASE_GAS;
+tx_base_gas(channel_create_tx, _)         -> ?TX_BASE_GAS;
+tx_base_gas(channel_force_progress_tx, Protocol) ->
+    case Protocol of
+        ?ROMA_PROTOCOL_VSN                -> ?TX_BASE_GAS;
+        ?MINERVA_PROTOCOL_VSN             -> ?TX_BASE_GAS;
+        _                                 -> 30 * ?TX_BASE_GAS
```
Comment (lucafavatella) [inline apps/aecore/src/aec_governance.erl]: I see same as call. Ok.

Comment (hanssv) [reply apps/aecore/src/aec_governance.erl]: That makes sense I think!

File: apps/aecore/src/aec_governance.erl [hunk 2]
```diff
@@ -84,29 +85,38 @@ block_mine_reward(Height) when is_integer(Height), Height > 0 ->
...
+-spec tx_base_gas(aetx:tx_type(), aec_hard_forks:protocol_vsn()) ->
+    non_neg_integer().
+tx_base_gas(contract_call_tx, _)          -> 30 * ?TX_BASE_GAS;
+tx_base_gas(contract_create_tx, _)        -> 5 * ?TX_BASE_GAS;
+tx_base_gas(spend_tx, _)                  -> ?TX_BASE_GAS;
+tx_base_gas(channel_deposit_tx, _)        -> ?TX_BASE_GAS;
+tx_base_gas(channel_close_mutual_tx, _)   -> ?TX_BASE_GAS;
+tx_base_gas(channel_close_solo_tx, _)     -> ?TX_BASE_GAS;
+tx_base_gas(channel_create_tx, _)         -> ?TX_BASE_GAS;
+tx_base_gas(channel_force_progress_tx, Protocol) ->
```
Comment (hanssv) [inline apps/aecore/src/aec_governance.erl]:
```
tx_base_gas(channel_force_progress_tx, Vsn) when Vsn < ?FORTUNA_PROTOCOL_VSN -> ?TX_BASE_GAS;
tx_base_gas(channel_force_progress_tx, Vsn) when Vsn >= ?FORTUNA_PROTOCOL_VSN -> 30 * ?TX_BASE_GAS;
```

is more readable?

**Pull request merged**